tfjs-layers/src/initializers.ts

```diff
@@ -545,9 +545,8 @@ export class Orthogonal extends Initializer {
 
   apply(shape: Shape, dtype?: DataType): Tensor {
     return tidy(() => {
-      if (shape.length !== 2) {
-        throw new NotImplementedError(
-            'The Orthogonal Initializer does not support non-2D shapes yet.');
+      if (shape.length < 2) {
```

Comment:
I modified this check since the default recurrent initializer in ConvLSTM2DCell is orthogonal which needs to support 4D. 

This update seems to be inline too with Python implementation (link below). 

Currently, I have only run unit tests in layer repo. Do you have any concerns?

https://github.com/tensorflow/tensorflow/blob/6016e8009ff8d9b3afa5ae5bafd052ed51d22f74/tensorflow/python/ops/init_ops.py#L571